src/Microsoft.Crank.Models/Microsoft.Crank.Models.csproj

```diff
@@ -2,12 +2,12 @@
 
   <PropertyGroup>
     <Description>Class to transfer data to/from the benchmark server.</Description>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Is this a project that gets used anywhere else outside the repo? Mainly asking as this is a change that is obviously breaking.

Reply:
Only in this repos, not published on NuGet.